key-spacing fine-grained control issue

eslint v2.3.0

Rule:

```
"key-spacing": [1, {
    "singleLine": {
        "beforeColon": false, "afterColon": true, "mode": "strict"
    },
    "multiLine": {
        "beforeColon": false, "afterColon": true, "mode": "minimum"
    }
}],
```

Code:

```js
module.exports = {
    foobar: 42,
    bat:    2 * 2
};
```

Expected result: no warnings.

Actual:
```
/home/dp/Projects/sdk/spa/plugin-css/index.js
  10:13  warning  Extra space before value for key 'bat'  key-spacing

✖ 1 problem (0 errors, 1 warning)
```

At the same time rule

```
"key-spacing": [1, {"beforeColon": false, "afterColon": true, "mode": "minimum"}],
```

gives no warnings
seems `multiLine` option is not working properly

--- a/lib/rules/key-spacing.js
+++ b/lib/rules/key-spacing.js
@@ -370,7 +370,7 @@ module.exports = function(context) {
 
         return {
             "Property": function(node) {
-                verifySpacing(node, isSingleLine(node) ? singleLineOptions : multiLineOptions);
+                verifySpacing(node, isSingleLine(node.parent) ? singleLineOptions : multiLineOptions);
             }
         };
 

--- a/tests/lib/rules/key-spacing.js
+++ b/tests/lib/rules/key-spacing.js
@@ -479,6 +479,25 @@ ruleTester.run("key-spacing", rule, {
                 "beforeColon": false
             }
         }]
+    }, {
+        code: [
+            "var obj = {",
+            "    foobar: 42,",
+            "    bat:    2",
+            "};"
+        ].join("\n"),
+        options: [{
+            singleLine: {
+                beforeColon: false,
+                afterColon: true,
+                mode: "strict"
+            },
+            multiLine: {
+                beforeColon: false,
+                afterColon: true,
+                mode: "minimum"
+            }
+        }]
     }],
 
     invalid: [{

